### ISSUE TITLE
[MU4 Issue] Accessibility: missing screenreader feedback after paste and other commands

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#16091]
**Describe the bug**
In Mu3, the paste command used to read status info - a description of the newly pasted selection - after the operation.  It does not in MU4, however.

**To Reproduce**
1. Enable a screen reader (tested with NVDA)
2. Select a range
3. Ctrl+C
4. Select a destination
5. Ctrl+V
6. See error: no feedback about the new selection to indicate the operation completed

**Platform information**
 - OS: Windows

**Additional context**
Actually quite a few commands don't report any real feedback, which is a shame - it would be great if pretty much every command would read aloud its own name (except for commands that result in something being read anyhow).  But I understand that would likely require more architectural changes.  Still, the case of paste is especially odd since normally what triggers the screen reader is a change in selection AFAIK, and a paste *does* result in a new selection (the pasted passage).

**Comment by jmwmusic** [comment]
In Word, Excell and some other programmes NVDA announces Cut, Copy & Paste when these actions are performed.  This would certainly be useful for NVDA to report.  JAWS apparently has a setting to report those every time the key stroke is performed regardless of whether the action took place, not so helpful!
